apps/idos-enclave/src/pages/methods/Password.tsx

```diff
@@ -1,43 +1,66 @@
+import { decode, encode } from "@stablelib/base64";
```

Comment:
So, this violates the current importing convention, which would dictate this to be:
```suggestion
import * as Base64Codec from "@stablelib/base64";
```

Did you decide to move away from that for tree-shaking purposes? Maybe something else?

Reply:
Yeah, these bare names really threw me off. :x

Can we follow up on this (in a different PR) to homogenize the import style? Zero rush, but I think it helps a lot.